roles/hsm_prep/README.md

```diff
@@ -0,0 +1,66 @@
+  that certificate will be retrieved instead.
+* The certs will be retrieved and stored in a secret (cifmw_hsm_luna_cert_secret)
+* As input to this role, we will require the ssh connection details and credentials for this VM.
+
+A minimal (one that takes the defaults) invocation of this role would be:
+
+- name: Set up Luna
+  ansible.builtin.include_role: hsm_prep
+  vars:
+    cifmw_hsm_admin_password: "<HSM admin password>"
+    cifmw_hsm_server_ip: "IP of HSM"
+    cifmw_hsm_client_ip: "IP of the client - this could be the hypervisor where the Openshift nodes run"
+    cifmw_hsm_luna_clientvm_ip: "IP for the pre-configured running lunaclient"
+    cifmw_hsm_luna_clientvm_user: "User to log into the pre-configured running lunaclient"
+    cifmw_hsm_luna_clientvm_password: "password to log into the pre-configured running lunaclient"
+    cifmw_hsm_luna_partition: "HSM partition for th client to join"
+
+## Parameters
+
+### HSM Details
+* `cifmw_hsm_hsmtype`: (String) The type of HSM required.  Currently, only "luna" is supported. Default value: `luna`
+* `cifmw_hsm_admin_user`: (String) The user to log into the HSM.  Default value: `admin`
+* `cifmw_hsm_admin_password`: (String) The password to log into the HSM.
+* `cifmw_hsm_server_ip`: (String) ip address or hostname of the HSM
+
+### Barbican Image and Details to get buildah script
+* `cifmw_hsm_barbican_operator_repo`: (String) Repo for barbican-operator. Default value: "https://github.com/openstack-k8s-operators/barbican-operator.git"
+* `cifmw_hsm_barbican_operator_version`: (String) Version for barbican-operator.  Default value: "main"
+* `cifmv_hsm_barbican_image_namespace`: (String) Namespace for barbican-operator source image.  Default value: "podified-antelope-centos9"
+* `cifmw_hsm_barbican_image_tag`: (String) Tag for barbican-operator source image.  Default value: "current-podified"
+
+### Role Parameters
+* `cifmw_hsm_cleanup`: (Boolean) Delete all resources created by the role at the end of the testing. Default value: `false`
+* `cifmw_hsm_working_dir`: (String) Working directory to store artifacts.  Default value: `/tmp/hsm-prep-working-dir`
+* `cifmw_hsm_client_ip`: (String) ip address or hostname of the client VM
+
+### Luna Parameters
+* `cifmw_hsm_luna_clientvm_ip`: (String) ip address or hostname of the luna client VM
+* `cifmw_hsm_luna_clientvm_user`: (String) user login for the luna client VM
+* `cifmw_hsm_luna_clientvm_password`: (String) user password for the luna client VM
+* `cifmw_hsm_luna_minclient_src`: (String) Location of linux minimal client tarball on the luna client VM. Default value: `/opt/data/Linux-Minimal-Client.tar.gz`
+* `cifmw_hsm_luna_binaries_src`: (String) Location of the luna binaries on the luna client VM. Default value: `/opt/data/bin`
+* `cifmw_hsm_luna_server_cert_src`: (String) Location of HSM server cert on the luna client VM.  Default value: ` /usr/safenet/lunaclient/bin/`
```

Comment:
Why put the cert in a `bin` folder?  It makes more sens t put it in `/opt/data` with the rest of the pre-provisioned files.